README.rst

```diff
@@ -113,3 +113,49 @@ A Simple Example
   print(view['x'])  # -> 1
   print(view.prefixes)  # -> ('ns_', '')
   print(view.keys())  # -> ['n_chans', 'no_change', 'ns_x']
+
+Asynchronous interface
+----------------------
+There is also an interface that works with asyncio. Use
+``katsdptelstate.aio.TelescopeState`` instead of
+``katsdptelstate.TelescopeState``. Functions that interact with the database are now
+coroutines.
+
+There are a few differences from the synchronous version, partly necessary due
+to the nature of asyncio and partly to streamline and modernise the code:
+
+- The constructor only takes a backend, not an endpoint. See below for an
+  example of how to construct a redis backend.
+- There is currently no support for reading or writing RDB files; you'll need
+  to create a synchronous telescope state client that connects to the same
+  storage.
+- There is no support for attribute-style access.
+- Item-style access is supported for read (``await ts.get('key')``), but not
+  for write. Use ``await ts.set('key', 'value')`` instead to set immutable
+  keys.
+- Instead of ``key in ts``, use ``await ts.exists(key)``.
```

Comment:
Why no `__setitem__` or `__contains__`?

Reply:
They can't return awaitables (`__contains__` has to return a boolean, and `__setitem__` can't usefully return anything), so `await (key in ts)` or `await ts['key'] = 'value` wouldn't be valid code.

Reply:
Thought as much!